docs/reference/elements/py-config.md

```diff
@@ -30,11 +31,14 @@ Reminder: when using TOML, any Arrays of Tables defined with double-brackets (li
 </py-config>
 ```
 
-- JSON config using the `type` attribute.
+### JSON config using the `type` attribute.
+
 ```html
 <py-config type="json">
   {
-    "autoclose_loader": true,
+    "splashscreen": {
+      "autoclose": false
```

Comment:
this should be set to `true` since we are showing an equivalence between TOML and JSON here

Reply:
and in the TOML based example which precedes this, we use `true`

Reply:
also fixed in #1022 though